.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,36 @@
+---
+name: "Bug report \U0001F41B" 
+about: "If something isn't working as expected \U0001F914."
+title: ''
+labels: bug
+assignees: J535D165, qubixes
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug with the front-end is.
+
+**To Reproduce**
+Steps to reproduce the problem:
+
+1.
+2.
+3.
+
+
+```
+your input code here
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Version information (please complete the following information):**
```

Comment:
We want the complete all the information, no? So probably not necessary to put it here.